config/cli.go

```diff
@@ -59,6 +59,8 @@ type Cli struct {
 	GateURL                   string
 	StreamHealthHookURL       string
 	BroadcasterURL            string
+	CdnRedirectPrefix         string
```

Comment:
As in some other comment below, I think you could have here the `*url.URL`, like this:

```go
CdnRedirectURL *url.URL
```